backends-velox/src/test/scala/org/apache/gluten/execution/FallbackSuite.scala

```diff
@@ -106,15 +112,38 @@ class FallbackSuite extends VeloxWholeStageTransformerSuite with AdaptiveSparkPl
     }
   }
 
-  // java.lang.NullPointerException
-  ignore("fallback final aggregate of collect_list") {
+  test("fallback collect_list") {
+    withSQLConf(
+      GlutenConfig.EXPRESSION_BLACK_LIST.key -> "collect_list"
```

Comment:
Is it possible to only make partial fallback happen in the test? Maybe, needs a config to control.

Reply:
One of the following tests is for partial fallback:

https://github.com/apache/incubator-gluten/pull/5655/files/c40a5ad359fe14b030f85627e0d983a2848c8bd6#diff-493b5fa71a5befc5adcd67b51e47e802b41870287619a7b752bbf85f53b1880cR139

Would you mean that?

Reply:
Added a set of cases here https://github.com/apache/incubator-gluten/pull/5655/files#diff-742ce9027f0d774d324125a130d9993bad09d4ea5eb6d35d791b39bb57138c77R983-R1057 for better coverage.